pkg/metrics/metrics.jsx

```diff
@@ -257,6 +278,9 @@ class CurrentMetrics extends React.Component {
                                     console.warn("Invalid line in df:", s);
                                     return;
                                 }
+
+                                if (hide.indexOf(fields[0]) >= 0)
```

Comment:
This is quadratic, and the Enterprise will find a way to make that a problem. Let's use a hash table or Set for `hide`.

Reply:
Heh, for the few (there are not that many loop devices usually), the overhead of a Set will probably be bigger; but indeed it would look a bit cleaner, I'll do that.

Reply:
Rewritten to use Set.